### ISSUE TITLE
Chore -> Add Unit Test Cases

### ISSUE THREAD

**Issue by ekas** [jslovers/jslovers-official-website#25]

**Comment by Neha** [comment]
@ekas do we want just the initial setup in this issue?

We should encourage every developer to add test-cases with their PRs

**Comment by ekas** [comment]
Yes, Just the initial setup. And you are right, we could ask developers to add their test-cases.

**Comment by mukeshyadav** [comment]
@ekas @Neha May I take this on? Here's what I plan to do:

1. Introduce react-testing-library.
2. Add some tests to give other developers an idea of how to start.
3. Implement a pre-commit hook to ensure that developers can't push code without adding tests, or bypass the tests if not required by using specific comments inside the PR, e.g.,` <!-- Test not required -->.`

If you have any other suggestions, please let me know.

**Comment by Neha** [comment]
@mukeshyadav thank you for your inputs.

1. yes. 100%
2. yes. I will create issues to add a few test-cases
3. we can do this after we have a few test-cases ready.
